src/com/seafile/seadroid2/ui/activity/UnlockGesturePasswordActivity.java

```diff
@@ -96,6 +97,8 @@ public void onPatternDetected(List<LockPatternView.Cell> pattern) {
                 mLockPatternView
                         .setDisplayMode(LockPatternView.DisplayMode.Correct);
                 settingsMgr.setupGestureLock();
+                Intent intent = new Intent(UnlockGesturePasswordActivity.this, BrowserActivity.class);
+                startActivity(intent);
```

Comment:
We use `UnlockGesturePasswordActivity` in both `BrowserActivity` and `SeafilePathChooser` (https://github.com/haiwen/seadroid/blob/1.4.0/src/com/seafile/seadroid2/ui/activity/SeafilePathChooserActivity.java#L167). And in the future we may have other activities to be protected by the gesture lock. So it's not proper to always start a browser activity here.


Maybe a better way is to override the `onBackPressed` method of `UnlockGesturePasswordActivity`.